soc/arm/st_stm32/stm32l4/power.c

```diff
@@ -93,6 +93,7 @@ void _sys_pm_power_state_exit_post_ops(enum power_states state)
 	case SYS_POWER_STATE_SLEEP_3:
 #endif /* CONFIG_HAS_SYS_POWER_STATE_SLEEP_3 */
 		LL_LPM_DisableSleepOnExit();
+		LL_LPM_EnableSleep();
```

Comment:
Would you be able to check if the problem also applies to stm32wb series (the other series power enabled in zephyr)?

Reply:
It's the same cortex M4, so in my opinion, It must affect stm32wb series too but I don't have a board with STM32WB to test it.... Do you want me to make the same patch for stm32wb and integrate it into this commit?

Reply:
yes, please @jdascenzio, could you apply this for the stm32wb as well.
Both soc have similar power saving mgt, so `LL_LPM_EnableSleep();`  is applicable to power.c of the stm32wb on `_sys_pm_power_state_exit_post_ops`